src/style.css

```diff
@@ -177,6 +177,23 @@ a:hover {
   transition: all 0.3s ease-out;
 }
 
+.transition-btn {
+  transition: 0.3s;
+}
+
+.break-all {
+  word-break: break-all;
+}
```

Comment:
Shouldn't need this https://tailwindcss.com/docs/word-break

Reply:
noted